### PR TITLE
feat(translation): add mandarin Chinese translation

### PR DESCRIPTION
**Pull request by pallymore**
part of #439 

Hello,

I know this is not in the languages requested. 
I took a stab and finished the Chinese translation anyways since I'm interested in this project (have been looking for a good github client for a very long time) 

Feel free to make any comments, or suggestions (from other Mandarin speakers)

There is no official Mandarin support for Github - I used the official translation guide for `git` instead.
https://github.com/git/git/blob/master/po/zh_CN.po

### CODE REVIEW

File: src/locale/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import I18n from 'react-native-i18n';
 import { common } from 'config';
-import { en, fr, nl, pt, ptBr, tr, ru, eo, gl, pl, de, es } from './languages';
+import { en, fr, nl, pt, ptBr, tr, ru, eo, gl, pl, de, es, zhCN } from './languages';
```
Comment (andrewda) [inline src/locale/index.js]: Use a lowercase "n" => `zhCn`.

File: src/locale/languages/zh-cn.js [hunk 2]
```diff
@@ -0,0 +1,345 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const zhCN = {
```
Comment (andrewda) [inline src/locale/languages/zh-cn.js]: Lowercase "n"

**Comment by andrewda** [comment]
Also, run prettier to format your code (it should run when you `git commit` unless your environment is a bit funky). Running prettier will fix the travis build.

**Comment by pallymore** [comment]
hello @andrewda , thanks for your review.
I fixed `zhCN` to `zhCn` 
and did `npm run prettier` 
although it generated some changes in other files as well :/ 
i committed them anyways assuming you want all code to be prettified.

File: src/locale/languages/zh-cn.js [hunk 3]
```diff
@@ -0,0 +1,335 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const zhCn = {
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: @andrewda Please forgive me. I have never seen this kind of abbreviation. Normal usages are `zhCN/zhTW/enUS` and so on. What do you think? Maybe we should revert it as `zhCN` and update `ptBr` as `ptBR`.

File: src/locale/languages/zh-cn.js [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+    login: {
+      connectingToGitHub: '正在连接GitHub...',
+      preparingGitPoint: 'GitPoint准备中...',
+      cancel: '取消',
+      welcomeTitle: '欢迎来到GitPoint',
+      welcomeMessage: '完全免费，功能最强大的GitHub客户端!',
+      notificationsTitle: '通知设定',
+      notificationsMessage: '查看并设定你所有未读和参与的通知',
+      reposTitle: '仓库和用户',
+      reposMessage: '便捷的获取仓库、用户、以及组织的信息',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 便捷地获取仓库、用户以及组织的信息

File: src/locale/languages/zh-cn.js [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+      issuesTitle: '问题和合并请求',
+      issuesMessage: '查看问题、参与讨论、以及处理合并请求',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 查看问题、参与讨论以及处理合并请求

File: src/locale/languages/zh-cn.js [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+      publicEvent: {
+        action: '使得',
+        connector: '公开',
+      },
+      pullRequestEvent: '{{action}} 合并请求',
+      pullRequestReviewEvent: '{{action}} 合并请求审查',
+      pullRequestReviewCommentEvent: '{{action}} 了合并请求',
+      pullRequestReviewEditedEvent: '{{action}} 他们对合并请求的审查',
+      pullRequestReviewDeletedEvent: '{{action}} 他们对合并请求的审查',
+      releaseEvent: '{{action}} 发行',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 发布

File: src/locale/languages/zh-cn.js [hunk 7]
```diff
@@ -0,0 +1,335 @@
...
+      donate: '捐助我们',
+      title: '设置',
+      language: '语言',
+      privacyPolicy: '隐私条款',
+      signOut: '登出',
+    },
+    privacyPolicy: {
+      title: '隐私条款',
+      effectiveDate: '最近更新: 2017年7月15日',
+      introduction: '我们很高兴您选择使用GitPoint。您可以在本隐私条款中看到我们如何使用，以及不使用，我们用户数据的信息。',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 我们很高兴您选择使用GitPoint。您可以在本隐私条款中看到我们使用哪些和不使用哪些用户数据。

File: src/locale/languages/zh-cn.js [hunk 8]
```diff
@@ -0,0 +1,335 @@
...
+      userDataTitle: '用户数据',
+      userData1:
+        '我们不使用您任何GitHub的信息。在使用GitHub登录后，用户的OAuth令牌只保存在客户端。我们无法获取令牌信息。我们用户查看或者保存用户的访问令牌。',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 我们不会把您的GitHub信息用于任何事。授权登录后，用户的OAuth令牌只会被直接保存在客户端设备存储中。我们无法获取令牌信息。我们从不查看或保存用户的访问令牌。

File: src/locale/languages/zh-cn.js [hunk 9]
```diff
@@ -0,0 +1,335 @@
...
+      userData2:
+        '这意味着，我们不可能查看、使用、或者分享用户的GitHub数据。即便隐私数据可以被查看，我们也不会查看或者保存它。如果它被意外的保存，我们会使用安全抹除的方案立刻删除它。再次重申，我们设计使用的登录方法可以保证这种情形不会发生。',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 这意味着，我们不可能查看、使用，或者分享用户的GitHub数据。即便隐私数据可以被访问，我们也不会查看或者保存它。如果它被意外地保存，我们会立即使用安全的抹除方案删除它。再次重申，我们已经特别地建立了授权机制以保证这种情形不会发生。

File: src/locale/languages/zh-cn.js [hunk 10]
```diff
@@ -0,0 +1,335 @@
...
+      analyticsInfoTitle: '分析数据',
+      analyticsInfo1:
+        '目前我们使用Google Analytics和iTunes App Analytics来帮助我们分析GitPoint的流量和使用趋势。这些工具从您的设备上收集包含设备类型、版本、区域、来源等信息。这些信息并无法被用来识别单个用户或者获取任何个人信息。',
+      analyticsInfo2: '如果我们使用第三方平台来收集追溯栈、错误报告、或者其他分析用信息，我们将保证用户的信息均为匿名并且加密。',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 如果我们使用第三方平台来收集追溯栈、错误报告，或者更多的分析信息，我们将保证用户的信息均为匿名并且加密。

File: src/locale/languages/zh-cn.js [hunk 11]
```diff
@@ -0,0 +1,335 @@
...
+      openSourceTitle: '开源',
+      openSource1: 'GitPoint是开源软件，所有对本软件的历史贡献信息都会一直向外界公开。',
+      openSource2: '向本App提交的每次代码贡献都会经过我们的审查以防止任何人注入任何的恶意代码。',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 每次向本App提交的代码贡献都会经过我们的审查以防止任何人注入任何形式的恶意代码。

File: src/locale/languages/zh-cn.js [hunk 12]
```diff
@@ -0,0 +1,335 @@
...
+      contactTitle: '联系方式',
+      contact1: '感谢阅读我们的隐私条款。我们希望您如同我们享受开发GitPoint的过程一般享受使用它！',
+      contact2: '如果您对本条款或GitPoint有任何问题，请在我们的仓库页面提交：',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 如果您对本条款或GitPoint有任何问题，请提交问题到

File: src/locale/languages/zh-cn.js [hunk 13]
```diff
@@ -0,0 +1,335 @@
...
+      unfollow: '取消关注',
+      follow: '关注',
+    },
+    repositoryList: {
+      title: '仓库列表',
+    },
+    followers: {
+      title: '粉丝',
+      text: '粉丝',
+      followsYou: '你已关注',
```
Comment (chinesedfan) [inline src/locale/languages/zh-cn.js]: 建议统一用“您”。或者除了较正式的公告性质的隐私声明，别的普通界面处统一用“你”。

File: src/locale/languages/zh-cn.js (hunk 3 above)
Comment (pallymore) [reply src/locale/languages/zh-cn.js]: Yea, I think `zhCN/ptBR` is more common as `CN/BR/US` are country abbreviations. `zhCn` or `enUs` looks weird to me. @andrewda ?

**Comment by pallymore** [comment]
Thanks for the thorough review. @chinesedfan 

Yes, I wasn't sure if I should leave the GitHub/Git jargons as they are since they can be weird once translated to Chinese. I guess we could gather some feedback from the users to see if we should only translate common UI/Message texts.

**Pull request merged**